.github/workflows/dotnet-build-and-release.yml

```diff
@@ -1,6 +1,6 @@
 name: Build dotnet solution and potentially upload the release
 on:
-  workflow_call:
+   workflow_call:

```

Comment:
Spacing / indentation can affect the way workflows run sometimes, so you should remove this extra space

Reply:
removed extra space